### PR TITLE
feat: add definitions for rpc endpoints to get validator add/remove s…

### LINKED ISSUE #390: Add proto definitions for data-node query of validator add/remove signature
# Task Overview

The proto change needed for https://github.com/vegaprotocol/data-node/issues/490

**In order to** be able to query the signature bundles needs to add/remove validators from the multisig contract
**We will** add data-node endpoints 
**So that** people can get the sig bundles in a form that can then be easily submitted

### PR DESCRIPTION
**Pull request by wwestgarth**
closes #390

### CODE REVIEW

File: sources/data-node/api/v2/trading_data.proto [hunk 1]
```diff
@@ -42,6 +42,82 @@ service TradingDataService {
...
+
+message GetERC20MultiSigSignerRemovedBundlesRequest {
+    // The node id of the validator of which a signature bundle is required
+    string node_id = 1;
+
+    // The ethereum address of the validator which will submit the bundle
+    string submitter = 2;
+
+    // Pagination controls
+    Pagination pagination = 3;
```
Comment (jeremyletang) [inline sources/data-node/api/v2/trading_data.proto:96]: I don't think the pagination is necessary here as well as we should return only one bundle as it's filtered by both the node_id to be removed and the submitter of the bundle.

File: sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -42,6 +42,82 @@ service TradingDataService {
     // Gets all available intervals for a given market along with the corresponding candle id
     rpc GetCandlesForMarket(GetCandlesForMarketRequest) returns (GetCandlesForMarketResponse);
 
+    // Gets the signature bundles that add a particular validator to the multisig contract
+    rpc GetERC20MultiSigSignerAddedBundles(GetERC20MultiSigSignerAddedBundlesRequest) returns (GetERC20MultiSigSignerAddedBundlesResponse);
```
Comment (jeremyletang) [inline sources/data-node/api/v2/trading_data.proto:46]: These should be probably singular right?

File: sources/data-node/api/v2/trading_data.proto [hunk 3]
```diff
@@ -42,6 +42,82 @@ service TradingDataService {
...
+
+    // Gets the signature bundles that remove a particular validator to the multisig contract
+    rpc GetERC20MultiSigSignerRemovedBundles(GetERC20MultiSigSignerRemovedBundlesRequest) returns (GetERC20MultiSigSignerRemovedBundlesResponse);
```
Comment (jeremyletang) [inline sources/data-node/api/v2/trading_data.proto:49]: Same comment than line 46

**Review by jeremyletang — COMMENTED** [review]
Just a few comments about the APIs, the bundles datastructure looks correct though

File: sources/data-node/api/v2/trading_data.proto [hunk 4]
```diff
@@ -42,6 +42,82 @@ service TradingDataService {
...
+
+
+message GetERC20MultiSigSignerAddedBundlesRequest {
+
+    // The node id of the validator of which a signature bundle is required
+    string node_id = 1;
+
+    // Pagination controls
+    Pagination pagination = 2;
+}
```
[the remark this answers is not in the document]
Comment (wwestgarth) [reply sources/data-node/api/v2/trading_data.proto:61]: What about the case where a validator gets promoted to a tendermint-validator and we get a signature bundle to add it, then it gets demoted. But then a bit later it gets promoted again, and a new signature bundle is made? Do we not want to keep both, or is the idea to only keep the latest for each validator?

[the remark this answers is not in the document]
Comment (jeremyletang) [reply sources/data-node/api/v2/trading_data.proto:61]: ah right. I see your point. I'm wondering if we should add in there a filter for the epoch then (also in the bundle, not to be used, but to be informative). So we know for which epoch this bundle was valid? 

Or maybe the whole history of it is enough as well then yea.

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**